docs/vc_patch.md

```diff
@@ -0,0 +1,113 @@
+# Nintendo 2DS/3DS Virtual Console Patch
+
+The Nintendo Virtual Console for the 2DS and 3DS can play Game Boy Color ROMs bundled into a `.cia` (CTR Importable Archive) file. Creating a `.cia` file for a `.gbc` ROM file is outside the scope of this project.
+
+An optional file bundled in the `.cia` archive is a `.patch` file with the same name as the `.gbc` file, used to determine when and under what conditions to enable certain Virtual Console functionalities, such as the Wireless Link capabilities. It also allows arbitrary content in the ROM to be modified, such as for disabling the Game Boy Printer features of Pokémon Crystal.
+
+
+## Build pokecrystal11_vc.patch
+
+To build **pokecrystal11_vc.patch**:
+
+```bash
+make crystal11_vc
+```
+
+This will also create two ROM files, **pokecrystal11.gbc** and **pokecrystal11_vc.gbc**. The pokecrystal11_vc.gbc file has the patches already applied to it. Do *not* use this file; rename pokecrystal11_vc.patch to **pokecrystal11.patch** and bundle it together with pokecrystal11.gbc in a `.cia` file.
+
+
+## vc/pokecrystal11_vc.constants.asm
+
+The `.constants.asm` file is used to create a `.constants.sym` file. Typical `.sym` files only list the values of *labels* (ROM banks and addresses); this file is used to list *constants* that are needed by the patch template. Any constants that the patch template needs must be explicitly printed here with the `vc_const` macro.
```

Comment:
(opinion)
This is a generated file. It's ok to document it I guess, but it's not really something the user needs to be aware of. Document this in the code that generates it, I'd say.

Reply:
The .constants.asm file is commented, but I think it's useful to reiterate about it here, because someone making edits could easily miss adding different constants to the .asm file.

Reply:
Oh, I mistook this one for the generated one. Nevermind, then.